Source/SourceKittenFramework/File.swift

```diff
@@ -31,6 +31,9 @@ public struct File {
             lines = contents.lines()
         } catch {
             fputs("Could not read contents of `\(path)`\n", stderr)
+            // necessary to set contents & lines because of rdar://21744509
+            contents = ""
+            lines = []
```

Comment:
Yay for this being fixed in swift-next https://github.com/apple/swift/commit/f524802e129f17de16fa6f8c3f08e5bcf75cc581